docs/reference/ml/ml-shared.asciidoc

```diff
@@ -507,6 +507,357 @@ tag::data-frame-analytics-stats[]
 An array of statistics objects for {dfanalytics-jobs}, which are
 sorted by the `id` value in ascending order.
 
+//Begin analysis_stats
+`analysis_stats`::
+(object)
+An object containing statistical data about the analysis.
++
+.Properties of `analysis_stats`
+[%collapsible%open]
+====
+//Begin classification_stats
+`classification_stats`:::
+(object)
+An object containing statistical data about the {classanalysis}.
++
+.Properties of `classification_stats`
+[%collapsible%open]
+=====
+//Begin class_hyperparameters
+`hyperparameters`::::
+(object)
+An object containing the hyperparameters of the {classanalysis}.
```

Comment:
```suggestion
An object containing the parameters of the {classanalysis}.
```
Also the section is called "hyperparameters" not all of these parameters are estimated in a hyperparameter  optimization procedure. Hence, I would use the word "parameters" here.